### PR TITLE
never 2.1.8

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
-- Could NOT find BISON: Found unsuitable version "2.3", but required is at least "2.7.1" (found /usr/bin/bison)
-- Found FLEX: /usr/bin/flex (found version "2.5.35") 
```

File: Formula/never.rb [hunk 1]
```diff
@@ -20,10 +20,10 @@ class Never < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "6075cc666f6d3d9033692d0ccb7e802411af1ec7d514dbe8bb83ab1001bf3f1c"
   end
 
+  depends_on "bison" => :build
   depends_on "cmake" => :build
+  depends_on "flex" => :build
```
Comment (branchvincent) [inline Formula/never.rb:25]: why change `flex`

Comment (chenrui333) [reply Formula/never.rb:25]: it is just a build dependency, I feel like we use both the latest versions of bison and flex. what do you think?

Comment (carlocab) [reply Formula/never.rb:25]: Let's not add dependencies we don't need; this leads to extra CI time for those formulae.

Comment (chenrui333) [reply Formula/never.rb:25]: ok

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**